modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -670,7 +675,9 @@ public CompletableFuture<Void> startAsync(ComponentContext componentContext) {
 
             attemptsObtainLock = txCfg.attemptsObtainLock().value();
 
-            executorInclinedPlacementDriver.listen(PrimaryReplicaEvent.PRIMARY_REPLICA_EXPIRED, this::onPrimaryReplicaExpired);
+            if (!enabledColocation()) {
```

Comment:
Why is this needed?

Reply:
Because this local event triggers table replica stop and the partition destruction, but under the colocation track we shouldn't even listen this.